cmd/trcd/main.go

```diff
@@ -1,23 +1,27 @@
 package main
 
 import (
-	"encoding/json"
 	"flag"
-	"io"
+	"math/rand"
 	"net"
 	"os"
 	"path/filepath"
+	"time"
 
 	log "github.com/Sirupsen/logrus"
 	"github.com/rvolosatovs/turtlitto/pkg/api"
+	"github.com/rvolosatovs/turtlitto/pkg/trcapi"
+	. "github.com/rvolosatovs/turtlitto/pkg/trcapi/trctest"
 )
 
 var (
-	sock = flag.String("socket", filepath.Join(os.TempDir(), "trc.sock"), "Path to the unix socket")
+	sock   = flag.String("socket", filepath.Join(os.TempDir(), "trc.sock"), "Path to the unix socket")
+	useTCP = flag.Bool("tcp", false, "Use a tcp socket instead of a Unix socket")
+	silent = flag.Bool("silent", false, "Disables automatic sending of random state updates")
```

Comment:
Why would we want this?

Reply:
We do want to send things from TRC, we may introduce a way to configure that though